OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -425,6 +428,19 @@ Action InitAudioPanel(Widget panel)
 					Game.Sound.UnmuteAudio();
 			};
 
+			var muteBackgroundMusicCheckbox = panel.Get<CheckboxWidget>("MUTE_BACKGROUND_MUSIC");
+			var muteBackgroundMusicCheckboxOnClick = muteBackgroundMusicCheckbox.OnClick;
+			muteBackgroundMusicCheckbox.OnClick = () =>
+			{
+				muteBackgroundMusicCheckboxOnClick();
+
+				if (!musicPlaylist.AllowMuteBackgroundMusic)
+					return;
+
+				if ((musicPlaylist.CurrentSongIsBackground && musicPlaylist.IsBackgroundMusicMuted()) || (musicPlaylist.CurrentSongIsBackground))
```

Comment:
No need for the extra set of parenthesis after the `||`

Reply:
This can actually be simplified down to:
```csharp
if ((musicPlaylist.CurrentSongIsBackground))
```